packages/webapp/src/components/Card/NewReleaseCard/NewReleaseCard.jsx

```diff
@@ -1,36 +1,27 @@
 import Card from '../index';
 import { ReactComponent as Star } from '../../../assets/images/signUp/new_feature.svg';
 import { Semibold, Text } from '../../Typography';
-import styles from '../../Typography/typography.module.scss';
+import typography from '../../Typography/typography.module.scss';
+import styles from './styles.module.css';
 import { useTranslation } from 'react-i18next';
-import { colors } from '../../../assets/theme';
+import { APP_VERSION } from '../../../util/constants';
 
 export function NewReleaseCard({ style }) {
   const { t } = useTranslation();
   return (
-    <Card
-      color={'info'}
-      style={{
-        display: 'flex',
-        flexDirection: 'row',
-        gap: '16px',
-        alignItems: 'center',
-        padding: '16px',
-        ...style,
-      }}
-    >
+    <Card color={'info'} style={{ ...style }} className={styles.card}>
       <Star />
       <div>
-        <Semibold style={{ color: colors.teal700, marginBottom: '12px', lineHeight: '20px' }}>
-          {t('SIGNUP.LITEFARM_UPDATED')}
```

Comment:
Just to make sure, we shouldn't delete old translations, correct??

Reply:
Correct! That one should stay as long as we are "supporting" v3.5.2